JSDemos/Demos/Scheduler/CustomTemplates/Angular/app/app.component.html

```diff
@@ -51,30 +53,26 @@ <h3>DXCinema Show Times</h3>
   </div>
 
   <div *dxTemplate="let model of 'tooltip-template'">
-    <div class="movie-tooltip">
-      <img [src]="getMovieById(model.appointmentData.movieId).image" />
-      <div class="movie-info">
-        <div class="movie-title">
-          {{
-            getMovieById(model.appointmentData.movieId).text +
-              " (" +
-              getMovieById(model.appointmentData.movieId).year +
-              ")"
-          }}
-        </div>
-        <div>
-          {{
-            "Director: " + getMovieById(model.appointmentData.movieId).director
-          }}
-        </div>
-        <div>
-          {{
-            "Duration: " +
-              getMovieById(model.appointmentData.movieId).duration +
-              " minutes"
-          }}
+    <ng-container
+      *ngIf="
+        getMovieById
+          | apply : model.appointmentData.movieId : moviesData as movie
+      "
```

Comment:
Let's change formatting:

```suggestion
      *ngIf="(getMovieById | apply : model.appointmentData.movieId : moviesData) as movie"
```